cmd/lock-main.go

```diff
@@ -141,10 +141,6 @@ func mainLock(ctx *cli.Context) error {
 	console.SetColor("Mode", color.New(color.FgCyan, color.Bold))
 	console.SetColor("Validity", color.New(color.FgYellow))
 
-	// Parse encryption keys per command.
-	_, err := getEncKeys(ctx)
-	fatalIf(err, "Unable to parse encryption keys.")
-
```

Comment:
now sure how is this related.

Reply:
This is not related - however including in this PR as this code cleanup is so minor that it doesnt warrant a separate PR. Removing this piece of code because encryption keys are needed only if a Stat or Get operation is attempted on an SSE-C encrypted object. `mc lock` is at bucket level and doesn't require encryption keys.

Reply:
There are no encryption keys involved with that command.